ScalingCarousel/Classes/ScalingCarouselView.swift

```diff
@@ -205,9 +219,13 @@ fileprivate extension PrivateAPI {
          Add constraints for height and top, relative to the
          ScalingCarouselView
         */
-        invisibleScrollView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
-        invisibleScrollView.topAnchor.constraint(equalTo: topAnchor).isActive = true
-        
+        if scrollDirection == .horizontal {
+            invisibleScrollView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
+            invisibleScrollView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        } else {
+            invisibleScrollView.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
```

Comment:
To hide the invisibleScrollView's scrolling indicator, please add the following at line 196 above:

```
invisibleScrollView.showsVerticalScrollIndicator = false
```